### ISSUE TITLE
Conclusion of the neovim version

### ISSUE THREAD

**Issue by RU927** [doctorfree/nvim-lazyman#10]
ubuntu 22.04, zsh and bash
I ran the install_neovim.sh script with the package manager apt to see if it worked. Neovim as well as its dependencies were installed before I ran the script. All went well except for one thing:

Installing dependencies
	Using previously installed git
	Using previously installed curl
	Using previously installed jq
	Using previously installed tar
	Using previously installed unzip
	Using previously installed fzf
	Using previously installed wget
	Using previously installed xclip
	Using previously installed wl-clipboard
	Using previously installed gh
	Using previously installed lazygit
	Using previously installed zoxide
	Using previously installed ripgrep
./install_neovim.sh: line 908: ((: .1
1: syntax error: operand expected (wrong marker ".1
1")

Installing language servers and tools
	Using previously installed ccls

looking at line 908 and above I repeated these commands in the terminal and here is their output:

nvim --version | head -1 
NVIM v0.10.0-dev-184+g907018e54

nvim --version | head -1 | grep -o '[0-9]\.[0-9]'
0.1
0.0

Apparently this syntax error is due to my version. I don't remember exactly which installation I got this version with, because I've been experimenting a lot in the last days. Probably this version was achieved by installing the deb package compiled from the source code. I will specify later when I repeat this process.
In any case nothing critical, because obviously this can happen when neovim is installed in a non-standard way and it does not affect the work of the rest of the script.

I want to take this opportunity to thank you for adding apt package manager. Thank you. 
Good luck with everything you focus on.

**Comment by doctorfree** [comment]
Yes, now that Neovim 0.10 is the nightly build, `install_neovim.sh` needs to be able to detect if that is installed. Thanks for reporting this. I will come up with a better test for the installed Neovim version.

On a side note, I discovered that Ubuntu and other Debian based Linux distributions require the `python3-venv` package in order for Mason to be able to install some tools. I added this fix to the `install_neovim.sh` script yesterday. Apt based systems may wish to update with `lazyman -U` to get this and other fixes.

As an even further aside, Lazyman users who previously installed using Homebrew but now wish to use the native package manager (e.g. apt or dnf or pacman) may wish to remove the Homebrew installed packages first. If Homebrew was only used for Lazyman setup then all of the Homebrew packages and Homebrew itself can be removed by running the Lazyman script `~/.config/nvim-Lazyman/scripts/uninstall_brew.sh`.
